Userland/Applications/PartitionEditor/main.cpp

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2022, Samuel Bowman <sam@sambowman.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Applications/PartitionEditor/PartitionEditorWindowGML.h>
+#include <Applications/PartitionEditor/PartitionModel.h>
+#include <LibCore/DirIterator.h>
+#include <LibCore/System.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/ComboBox.h>
+#include <LibGUI/ItemListModel.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/MessageBox.h>
+#include <LibGUI/TableView.h>
+
+using namespace PartitionEditor;
+
+Vector<String> get_device_paths();
+
+Vector<String> get_device_paths()
```

Comment:
This can just be `static`

Reply:
TIL C++ has C style static functions outside of classes. Fixed.